src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageMetadataControl.xaml

```diff
@@ -315,67 +304,41 @@ Background="{DynamicResource {x:Static nuget:Brushes.DetailPaneBackground}}"
-                FontWeight="Bold"
-                x:Name="_targetFramework"
-                AutomationProperties.Name="{Binding Text}"
-                Visibility="{Binding TargetFramework,Converter={StaticResource NuGetFrameworkToVisibilityConverter}}" />
-              <ItemsControl
-                ItemsSource="{Binding Dependencies}"
-                IsTabStop="False"
-                AutomationProperties.LabeledBy="{Binding ElementName=_targetFramework}">
-                <ItemsControl.ItemTemplate>
-                  <DataTemplate>
-                    <TextBox
-                      x:Name="_dependencies"
-                      Style="{DynamicResource SelectableTextBlockStyle}"
-                      AutomationProperties.Name="{Binding Text}"
-                      Text="{Binding Mode=OneWay}"/>
-                  </DataTemplate>
-                </ItemsControl.ItemTemplate>
-              </ItemsControl>
-              <TextBlock
-                FontStyle="Italic"
-                Text="{x:Static nuget:Resources.Text_NoDependencies}"
-                x:Name="_noDependencies"
-                AutomationProperties.Name="{Binding Text}"
-                Visibility="{Binding Dependencies,Converter={StaticResource EmptyEnumerableToVisibilityConverter}}" />
-            </StackPanel>
-          </DataTemplate>
-        </ItemsControl.ItemTemplate>
-      </ItemsControl>
-
-      <TextBlock
-        Margin="0,8,0,0"
-        Visibility="{Binding Path=HasDependencies,Converter={StaticResource InvertedBooleanToVisibilityConverter}}"
-        FontStyle="Italic"
-        x:Name="_hasDependencies"
-        AutomationProperties.Name="{Binding Text}"
-        AutomationProperties.LabeledBy="{Binding ElementName=_dependenciesLabel}"
-        Text="{x:Static nuget:Resources.Text_NoDependencies}" />
-    </StackPanel>
+    <!-- dependencies treeview -->
+    <TreeView
+        x:Name="_dependencies"
+        Grid.Row="3"
+        Margin="-21,0,0,0"
+        BorderThickness="0"
+        Background="{x:Null}"
+        TreeViewItem.Selected="OnItemSelected"
+        LostFocus="TreeView_LostFocus"
+        AutomationProperties.Name="{x:Static nuget:Resources.Label_Dependencies}">
+      <TreeView.Template>
+        <!--
+          Replaces default control template for this treeview with a slimmer control template.
+          Specifically, this one has no ScrollViewer...which wasn't being used, and was making mousewheel not work for
+          the larget detail scrollable area while the mouse was over the treeview.
+        -->
+        <ControlTemplate TargetType="TreeView">
+          <ItemsPresenter />
+        </ControlTemplate>
+      </TreeView.Template>
+      <TreeView.Resources>
+        <Style TargetType="TreeViewItem">
+          <Setter Property="IsExpanded" Value="true" />
+          <Setter Property="AutomationProperties.Name" Value="" />
```

Comment:
Is this intentional ?

Reply:
This make the narrator reads each tree view item as the type of the tree view item: "NuGet.PackageManagement.UI.PackageDependencySetMetadata..."

![image](https://user-images.githubusercontent.com/1192347/74207845-fbd15880-4c35-11ea-835a-7ae0c3ad1dc6.png)

Possible fixes:

- Remove the line
- Provide a more friendly name, for example, `"Target Framework {tfm}, version {version}"`

Reply:
Will dig into this behavior and options. Looks to me like it is exposing some details about the data object. the other data is already exposed naturally in the content under it (header and children), so I'd prefer not to repeat.

Reply:
fixed with workaround...and opened a WPF bug: https://github.com/dotnet/wpf/issues/2552